zcash_keys/src/keys.rs

```diff
@@ -1506,7 +1509,10 @@ mod tests {
             }
 
             let ua = ufvk
-                .address(d_idx, UnifiedAddressRequest::unsafe_new(false, true, true))
+                .address(
+                    d_idx,
+                    Some(UnifiedAddressRequest::unsafe_new(false, true, true)),
```

Comment:
Huh, I didn't realize that these test vectors don't include Orchard. Non-blocking.